src/modules/dashboard/transformers.ts

```diff
@@ -0,0 +1,90 @@
+import { AddressZero, HashZero } from 'ethers/constants';
+
+import { SubgraphActions, TransactionsMessagesCount } from '~data/index';
+import { ColonyActions, FormattedAction } from '~types/index';
+
+export const getActionsListData = (
+  unformattedActions?: SubgraphActions,
+  transactionsCommentsCount?: TransactionsMessagesCount,
+): FormattedAction[] => {
+  let formattedActions = [];
+  Object.keys(unformattedActions || {}).map((subgraphActionType) => {
+    formattedActions = formattedActions.concat(
+      (unformattedActions || {})[subgraphActionType].map(
+        (unformattedAction) => {
+          const formatedAction = {
+            id: unformattedAction.id,
+            actionType: ColonyActions.Generic,
+            initiator: unformattedAction.agent,
+            recipient: AddressZero,
+            amount: '0',
+            tokenAddress: AddressZero,
+            symbol: '???',
+            decimals: '18',
+            fromDomain: '1',
+            toDomain: '1',
+            transactionHash: HashZero,
+            createdAt: new Date(),
+            commentCount: 0,
+          };
+          const {
+            transaction: {
+              hash,
+              block: { timestamp },
+            },
+          } = unformattedAction;
+          const transactionComments =
+            /*
+             * @NOTE Had to disable this as prettier was being too whiny
+             * and all suggestions it made broke the style rules
+             *
+             * This sadly happens from time to time...
+             */
+            // disable-next-list prettier/prettier
+            transactionsCommentsCount?.colonyTransactionMessages?.find(
+              ({ transactionHash }) => transactionHash === hash,
+            );
+          if (subgraphActionType === 'oneTxPayments') {
```

Comment:
We'll have types for all actions we support in the app.

This will need to be refactored once we have more types. (I think I've already added an issue for this)